css/properties/font-smooth.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "font-smooth": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-smooth",
+          "support": {
+            "chrome": {
+              "version_added": "5"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "25",
+              "notes": "A similar property is implemented in Firefox called <code>-moz-osx-font-smoothing</code>. It only works on Mac OS X/macOS."
```

Comment:
This should be represented differently:

```json
firefox: [
  { 
    "version_added": "25"
  },
  {
    "version_added": true,
    "alternative_name": "-moz-osx-font-smoothing",
    "notes": "Only works on Mac OS X / macOS."
  }
]
```

Reply:
Thanks, fixed in 31b131f609976290e18e64bb04ed8fd899318223.

PS: The `"version_added": "25"` seems refer to `-moz-osx-font-smoothing` (see [bug 857142](https://bugzilla.mozilla.org/show_bug.cgi?id=857142)).

Reply:
Great finding! :+1: